init.sh

```diff
@@ -39,5 +39,6 @@ for channel in stable beta nightly; do
 	[[ -d root-$channel ]] && mv root-$channel root-${channel}.old
 	mv root-${channel}.new root-$channel
 	[[ -d root-${channel}.old ]] && rm -rf root-${channel}.old
+	arch-chroot root-$channel cargo install --root /usr --vers 0.3 rustfmt
```

Comment:
Eh this is using quite a small surface area of rustfmt so it seems like the breakage window is unlikely? We could always update manually if anything goes awry